src/app/calls.ts

```diff
@@ -0,0 +1,71 @@
+import {AppCallResponse, AppSelectOption} from 'mattermost-redux/types/apps';
+
+class Calls {
+    // createForm returns a form response to create a ticket from a post
+    createForm(message: string): AppCallResponse {
```

Comment:
I'm thinking this should exist in a file `src/forms/create_ticket.ts`

Reply:
Agreed. Done.